script/collect.js

```diff
@@ -118,11 +118,14 @@ async function main () {
   const oldNpmCount = old.find(release => release.npm_package_name === 'electron').length
   const newNpmCount = releases.find(release => release.npm_package_name === 'electron').length
 
+  const diffNotesCount = old.some((release, index) => release.body !== releases[index].body)
```

Comment:
This logic is assuming that the order of releases is consistent, this logic should probably identify the matching `release` from `old` in a deterministic way rather than assuming the order of releases in the array is the same.

```js
const releaseBodyChanged = old.some((oldRelease) => {
  const newRelease = releases.find(release => release.version === oldRelease.version)
  return !newRelease || newRelease.body !== oldRelease.body
})
```